rffmpeg

```diff
@@ -839,13 +882,6 @@ if __name__ == "__main__":
 
     if "rffmpeg" in cmd_name:
         run_control(config)
-    else:
-        if not Path(config["db_path"]).is_file():
```

Comment:
There should almost certainly still be a check here, otherwise the process might run with no database connection and cause some issues. Perhaps 

```
if DB_TYPE == "SQLITE" and not Path...
```

Reply:
But it would fail correctly as soon as it calls `dbconn(config)`, I just moved that check from `main` to `dbconn(config)`. If that's not enough I'll make the change.

Reply:
Oh you're completely right, that makes more sense.